addons/calendar/tests/test_mail_activity_mixin.py

```diff
@@ -48,7 +48,7 @@ def schedule_meeting_activity(record, date_deadline, calendar_event=False):
             meeting.calendar_event_id = calendar_event
             return meeting
 
-        group_partner_manager = self.env['ir.model.data'].xmlid_to_res_id('base.group_partner_manager')
+        group_partner_manager = self.env['ir.model.data']._xmlid_to_res_id('base.group_partner_manager')
         self.user_employee.write({
             'tz': self.user_admin.tz,
             'groups_id': [Command.link(group_partner_manager)]
```

Comment:
Might be better to use `ref` here, in case the the record behind the xid has been deleted (but the xid is still there) it would trigger either a weird error or a weird no-op.